yts/yts.py

```diff
@@ -13,13 +13,29 @@
 class YTS:
     """ provides an API for downloading yts yifi movies. """
 
-    # Initialize with the current yts url
     def __init__(self, yts_url):
+        """
+        Initialize with the current yts url 
+
+        Parameters
+        ----------
+            yts_url: str, required
```

Comment:
It would be better if the parameters and return types were unindented.
```
Parameter
--------------
yts_url: str, required
         The URL of the yts website
```
You can go through the [numpy doc style](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html) if you'd like. : )